### ISSUE TITLE
3.12.3 incompatibility

### ISSUE THREAD

**Issue by SheepTank** [JoeJoeTV/AstroTuxLauncher#10]
Ubuntu 24.04.3 LTS
python3.13-venv

```
(.venv) sheeptank@astroneer:~/AstroTuxLauncher$ python3 AstroTuxLauncher.py -p ../astroneer_server/ start
/home/sheeptank/AstroTuxLauncher/AstroTuxLauncher.py:42: SyntaxWarning: invalid escape sequence '\{'
  {ansi.BLUE}/ /| | / ___/ __/ ___/ __ \{ansi.YELLOW}/ / / / / / |/_/
/home/sheeptank/AstroTuxLauncher/AstroTuxLauncher.py:45: SyntaxWarning: invalid escape sequence '\_'
  """
/home/sheeptank/AstroTuxLauncher/AstroTuxLauncher.py:45: SyntaxWarning: invalid escape sequence '\_'
  """

    ___         __           ______
   /   |  _____/ /__________/_  __/_  ___  __
  / /| | / ___/ __/ ___/ __ \/ / / / / / |/_/
 / ___ |(__  ) /_/ /  / /_/ / / / /_/ />  <
/_/  |_/____/\__/_/   \____/_/  \__,_/_/|_|
               L a u n c h e r

Unofficial Astroneer Dedicated Server Launcher for Linux
v1.1.8

[20:12:18] [Launcher/INFO] Configuration file path: /home/sheeptank/AstroTuxLauncher/launcher.toml
Error while initializing launcher on line 571: TypeError: replace() takes 1 positional argument but 2 were given
Traceback (most recent call last):
  File "/home/sheeptank/AstroTuxLauncher/AstroTuxLauncher.py", line 571, in <module>
    launcher = AstroTuxLauncher(args.config_path, args.astro_path, args.depotdl_exec, force_debug_log=args.log_debug)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sheeptank/AstroTuxLauncher/AstroTuxLauncher.py", line 200, in __init__
    self.config = dataclasses.replace(self.config, {"AstroServerPath": astro_path})
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: replace() takes 1 positional argument but 2 were given

Quitting...
```

**Comment by SheepTank** [comment]
Further info

`[20:32:26] [Launcher/INFO] Updating Astroneer Dedicated Server app from Steam...
Updating app 728470 in 2:32.2 
[20:34:58] [Steam/INFO] Finished updating app 728470 in 152.15 seconds
[20:34:58] [Launcher/INFO] Sucessfully updated Astroneer Dedicated Server to version 1.31.79.0
[20:35:29] [Launcher/ERROR] Error occured during updating of wine prefix: Command '['/usr/bin/wine', 'wineboot']' timed out after 30 seconds
[20:35:29] [Launcher/INFO] Quitting... (Reason: Error while updating WINE prefix)`

**Comment by SheepTank** [comment]
```
BANNER_LOGO = f"""{ansi.weight.bold}
    {ansi.BLUE}___         __           {ansi.YELLOW}______          
   {ansi.BLUE}/   |  _____/ /__________{ansi.YELLOW}/_  __/_  ___  __
  {ansi.BLUE}/ /| | / ___/ __/ ___/ __ \\{ansi.YELLOW}/ / / / / / |/_/
 {ansi.BLUE}/ ___ |(__  ) /_/ /  / /_/ {ansi.YELLOW}/ / / /_/ />  <  
{ansi.BLUE}/_/  |_/____/\\__/_/   \\____{ansi.YELLOW}/_/  \\__,_/_/|_|  {ansi.reset}
"""
BANNER_SUBTITLE = "L a u n c h e r".center(45)```

Cleared up some syntax warnings with some escapes.

**Comment by JoeJoeTV** [comment]
Fixed in bbc4c034eb96b6ef26001a00e894b2f734eb63f9
